src/controllers/moneycontroller.js

```diff
@@ -11,7 +12,13 @@ export default class MoneyController extends Controller
      */
     static updateMoney(amount)
     {
-        GAME.model.config.money = GAME.model.config.money + parseFloat(amount);
-        $("#money").html(GAME.model.config.money.toFixed(2));
+        let config = GAME.model.config;
+
+        const scoreController = new ScoreController();
```

Comment:
Do this after updating the money (this should ideally be the line directly before `updateScore`.